File: level1/4 - Lecture de l'entrée/8 - Bétail.c
//coded by cybereagle2001
#include <stdio.h>
#define repeat(nb) for(int _loop = 1, _max = (nb); _loop <= _max; _loop++)

int main()
{
   int totalKarvas = 0;
   repeat (20)
   {
      int nbBetes;
      scanf("%d", &nbBetes);
      totalKarvas = totalKarvas + nbBetes;
   }
   printf("%d\n", totalKarvas);
}

File: java/level 1/4-Lecture de l'entrée/8-Bétail.java
/**coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int x;
      int j=0;
      for (int loop=0; loop<20; loop++){
         x= input.nextInt();
         j= j+x;
      }
      System.out.println(j);
   }
}
